GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -103,6 +103,16 @@ protected override List<MapEntityMetadata> Parse()
                             metadata.BoundingBox1 = boundingBox;
                         }
                     }
+                    else if (modelName.StartsWith("Skill_HealingSpot"))
+                    {
+                        XmlNode blockCoord = node.SelectSingleNode("property[@name='Position']");
+                        CoordS healingCoord = CoordS.Parse(blockCoord?.FirstChild.Attributes["value"].Value ?? "0, 0, 0");
+                        if (healingCoord.Equals(CoordS.From(0, 0, 0)))
+                        {
+                            continue;
+                        }
+                        metadata.HealthSpot = healingCoord;
```

Comment:
Perhaps we should stick to either using `health` or `healing`, so maybe call this `HealingSpot`?